source/core/ut_suite_manager.pkb

```diff
@@ -434,6 +436,11 @@ create or replace package body ut_suite_manager is
             l_procedure_name := regexp_substr(l_path, '^\w+\.(\w+)(\.(\w+))?$', subexpression => 3);
 
             l_temp_suite := config_package(l_schema, l_package_name);
+            
```

Comment:
Maybe use dbms_assert.sql_object_name first, and if it fails, we know the object name is not valid.
Then do the if statement, and raise error stating, that the package is not a properly annotated unit test suite package.